test/mlir/affine_conversion/callback_ops.mlir

```diff
@@ -4,13 +4,29 @@
 
 // -----
 
+// Convbias Op
+// CHECK-LABEL: func @simple_convbias
+//       CHECK: %[[GA0:.*]] = llvm.mlir.addressof @{{[a-zA-Z_][a-zA-Z0-9_]*}} : !llvm<"{ i8, [3 x i64], [3 x i64], [3 x i64], [3 x i64] }*">
+//       CHECK: %[[C0:.*]] = constant {{[0-9]+}} : i64
```

Comment:
Use CHECK-DAG since there is no ordering constraints on the constant/cast ops ?

Reply:
done.